package.json

```diff
@@ -28,19 +28,21 @@
   },
   "homepage": "https://clickhouse.com",
   "scripts": {
-    "watch": "vitest --watch",
-    "test": "vitest",
-    "dev": "vite",
-    "generate-tokens": "node build-tokens.js",
-    "typecheck": "tsc --noEmit",
     "build": "tsc && vite build",
+    "build-storybook": "storybook build",
     "build:watch": "watch 'npm run build' ./src",
+    "chromatic": "npx chromatic",
+    "dev": "vite",
+    "generate-tokens": "node build-tokens.js",
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "prettify": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\" --config .prettierrc",
+    "prettier:check": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\" --config .prettierrc",
     "preview": "vite preview",
+    "publish:click-ui": "npm run test && npm run build && npm publish",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build",
-    "chromatic": "npx chromatic",
-    "publish:click-ui": "npm run test && npm run build && npm publish"
+    "test": "vitest",
+    "typecheck": "tsc --noEmit",
+    "watch": "vitest --watch"
```

Comment:
This change alphabetizes this script list and adds `prettify` and `prettier:check`